source/credentials_provider_default_chain.c

```diff
@@ -305,11 +306,12 @@ struct aws_credentials_provider *aws_credentials_provider_new_chain_default(
 #endif /* BYO_CRYPTO */
     }
 
-    enum { providers_size = 4 };
+    enum { providers_size = 5 };
     struct aws_credentials_provider *providers[providers_size];
     AWS_ZERO_ARRAY(providers);
     size_t index = 0;
 
+    /* Providers that touch fast local resources */
     struct aws_credentials_provider_environment_options environment_options;
```

Comment:
utterly trivial, make the comment look like it applies to more than just this 1 code block
```suggestion
    /* Providers that touch fast local resources... */
    
    struct aws_credentials_provider_environment_options environment_options;
```